### DISCUSSION TITLE
Comments?

### DISCUSSION THREAD

**Discussion by 241m** [jotaen/xit#46] [File Format]
Hello all!

How about adding comment lines in the [x]it spec? Lines that are ignored by any tools like parsers or editors. My main reason for suggesting this is for using [vim modelines](https://vim.fandom.com/wiki/Modeline_magic).

Something like:

```xit
# This is a comment
# It means nothing

TODO
[ ] Line 1
[ ] Line 2

# vim: filetype=xit:
```

As per the current spec, ~syntax highlighters~ parsers (at least [tree-sitter-xit](https://github.com/synaptiko/tree-sitter-xit)) treat them as title lines.

Might be similar/related to #10

**Comment by coutvv** [comment]
Hi! I have the same problem and I resolve it by editing the plugin's script.

I added the next line to your ~/.vim/pack/3rd/start/vim-xit/syntax/xit.vim
```Vimscript
syn region xitComment start="\/\/" end="$" contained
hi def link xitComment Comment 
```

And add this new region (xitComment) to all regions that could contained comment. 
For example in `xitCheckboxOpenDesc`:
```Vimscript
syn region xitCheckboxOpenDesc start="." end=/\v(\[|^[a-zA-Z0-9])/me=e-1 contained contains=xitTag,xitDueDate,xitComment
```

By the way I'm not an expert in the vim scripting (this is the first problem I have solved with vimscript) so be aware :)

Hope it'll help
